packages/turf-interpolate/index.js

```diff
@@ -55,7 +67,8 @@ function interpolate(points, cellSize, options) {
   gridType = gridType || "square";
   weight = weight || 1;
 
-  var box = bbox(points);
+  box = box || bbox(points);
```

Comment:
Better to use `??` here.

Reply:
I looked at the documentation and it mentioned that instead of using ES6, `??` is an ES2020 feature, so I used `||` , if ES6 can be used that would be great.
[Docs](https://turfjs.org/docs/contributing)

Reply:
Would say that documentation is well out of date, and we use `??` successfully in other places. Go for it.